node/npm_import_error.ts

```diff
@@ -0,0 +1,25 @@
+class NPMImportError extends Error {
+  constructor(originalError: Error, public readonly moduleName: string) {
+    super(
+      `It seems like you're trying to import an NPM module. This is only supported in Deno via CDNs like esm.sh, have you tried 'import ... from "https://esm.sh/${moduleName}"'?`,
```

Comment:
A couple of nits:

- I think "npm" is the correct capitalisation
- Changed `import ...` to `import mod` because that's a valid instruction, in case people try to copy/paste

```suggestion
      `It seems like you're trying to import an npm module. This is only supported in Deno via CDNs like esm.sh. Have you tried 'import mod from "https://esm.sh/${moduleName}"'?`,
```